octoprint_enclosure/__init__.py

```diff
@@ -157,14 +157,25 @@ def on_settings_migrate(self, target, current=None):
         self._logger.info("rpi_inputs: %s", self.rpi_inputs)
         self._logger.info("#########        End Current Settings        #########")
         if current >= 4 and target == 6:
-            self._logger.warn("######### migrating settings to v6 #########")
             old_outputs = self._settings.get(["rpi_outputs"])
+            old_inputs = self._settings.get(["rpi_inputs"])
+            self._logger.warn("######### migrating settings to v6+ #########")
             for rpi_output in old_outputs:
                 if 'shutdown_on_failed' not in rpi_output:
                     rpi_output['shutdown_on_failed'] = False
                 if 'shell_script' not in rpi_output:
                     rpi_output['shell_script'] = ""
+
+            if target >= 7:

```

Comment:
This `if target >= 7` block will never execute, because the previous block is `if current >= 4 and target == 6:`

`target` can't be equal to 6 and >= 7 at the same time...

Reply:
Thanks for looking at this! :) 
I just pushed a fix for this, it should of course be "target >= 6": https://github.com/vitormhenrique/OctoPrint-Enclosure/pull/392/commits/77ac49ecf98bfef2b3fac9ac2f7a51edf2f5276f

That was an oversight when preparing the branch for the PR, I had the upgrade tested and working in my "main" branch:
https://github.com/Dak0r/OctoPrint-Enclosure-with-Custom-Graphs/blob/master/octoprint_enclosure/__init__.py#L132

Sorry for that